script/react

```diff
@@ -5,7 +5,7 @@ if [[ "$unamestr" == 'Darwin' ]]; then
   #MacOS
   sed -i ".original" "/gem 'react-rails'/d" "Gemfile"
   sed -i ".original" '$!N;/^\n$/!P;D' "Gemfile"
-  sed -i ".original" "s/gem 'webpacker'\(, '~> [0-9]\(\.[0-9]\(\.[0-9]\)\?\)\?'\)\?/&\n\ngem 'react-rails'/g" "Gemfile"
+  sed -i ".original" $'s/gem \'webpacker\'.*$/&\\\n\\\ngem \'react-rails\'/g' "Gemfile"
   bundle install
   rails webpacker:install:react
   sed -i ".original" "s/body$/  = javascript_pack_tag 'application'\n  &/g" "app/views/layouts/application.html.slim"
```

Comment:
@mtejedorwolox check this line. It has the same problem as before. Only one step more and 👍